src/k-provider/common/base-provider.ts

```diff
@@ -1,52 +1,52 @@
-// @flow
-import {getLogLevel, setLogLevel, setLogger, type LogLevelType, LogLevel} from '../../util/logger';
+import {getLogLevel, setLogLevel, setLogger, LogLevelType, LogLevel} from '../../util/logger';
 import DataLoaderManager from './data-loader-manager';
 import Error from '../../util/error/error';
+import {ProviderEntryListObject, ProviderMediaConfigObject, ProviderPlaylistInfoObject, ProviderMediaConfigSourcesObject, ProviderNetworkRetryParameters, ProviderOptionsObject, ProviderPlaylistObject} from '../../types';
 
 export default class BaseProvider<MI> {
-  _partnerId: number;
-  _widgetId: ?string;
-  _ks: string;
-  _uiConfId: ?number;
-  _dataLoader: DataLoaderManager;
-  _playerVersion: string;
-  _logger: any;
-  _isAnonymous: boolean;
-  _networkRetryConfig: ProviderNetworkRetryParameters = {
+  private _partnerId: number;
+  private _widgetId?: string;
+  private _ks: string;
+  private _uiConfId?: number;
+  public _dataLoader!: DataLoaderManager;
+  private _playerVersion: string;
+  public _logger: any;
+  protected _isAnonymous: boolean;
+  public _networkRetryConfig: ProviderNetworkRetryParameters = {
```

Comment:
In terms of convention, an underline under a word is unnecessary, but at this stage I preferred not to touch it because it is dangerous since you have to go through all the places that access it from the outside